logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java

```diff
@@ -104,223 +100,155 @@
  *         &lt;param name=&quot;Subject&quot; value=&quot;Last Event: %statusCode %requestURL&quot; /&gt;
  *         &lt;param name=&quot;To&quot; value=&quot;server_admin@domain.org&quot; /&gt;
  *      &lt;/appender&gt;
- *      &lt;appender-ref ref=&quot;SMTP&quot; /&gt;
+ *      &lt;appender-ref ref=&quot;SMTP&quot; /&gt; 
  *    &lt;/configuration&gt;
  * </pre>
- *
+ * 
  * @author Ceki G&uuml;lc&uuml;
  * @author S&eacute;bastien Pennec
  */
 public class RequestLogImpl extends ContextBase implements RequestLog,
-        AppenderAttachable<IAccessEvent>, FilterAttachable<IAccessEvent> {
+    AppenderAttachable<AccessEvent>, FilterAttachable<AccessEvent> {
 
   public final static String DEFAULT_CONFIG_FILE = "etc" + File.separatorChar
-          + "logback-access.xml";
+      + "logback-access.xml";
 
-  AppenderAttachableImpl<IAccessEvent> aai = new AppenderAttachableImpl<IAccessEvent>();
-  FilterAttachableImpl<IAccessEvent> fai = new FilterAttachableImpl<IAccessEvent>();
-  String fileName;
-  String resource;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2372229938/issues/source?bid=3222618&fileBranchId=3222692#l117)